lib/gui/models/flash-state.js

```diff
@@ -220,3 +220,20 @@ exports.getLastFlashSourceChecksum = () => {
 exports.getLastFlashErrorCode = () => {
   return exports.getFlashResults().errorCode;
 };
+
+/**
+ * @summary Get last flash uuid
+ * @function
+ * @public
+ *
+ * @description
+ * This function returns undefined if there was no last flash.
```

Comment:
Should this mention that this will also be undefined after `resetState` ?
(apologies for the confusing questions, but I initially misunderstood how this worked)